esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -551,6 +645,75 @@ static inline bool isPublishedQuery(EsdlMethodImplType implType)
     return (implType==EsdlMethodImplRoxie || implType==EsdlMethodImplWsEcl);
 }
 
+void EsdlServiceImpl::handleFeatureAuthorization(IEspContext& context, IEsdlDefMethod& esdlDef)
+{
+    auto method = esdlDef.queryName();
```

Comment:
Convince me that in these places auto is helpful and not just a shortcut that isn't self documenting.